script.js

```diff
@@ -0,0 +1,8 @@
+$(document).ready(function(){
+
+    $("#first-name").text("Jordan");
+    $("#last-name").text("Sutherland");
+    $(".mcc").css( "font-style", "italic");
+    $(".mcc").css( "color", "blue");
+    $("#hobby").append("<li>Video Games</li><li>Art</li>")
```

Comment:
Very nice use of jQuery to target elements by ID or class and then add/append the content or style accordingly! Well done! This is exactly what I would hope to see.